opencti-platform/opencti-front/src/utils/tests/filtersUtils.test.tsx

```diff
@@ -0,0 +1,47 @@
+import { describe, it, expect } from 'vitest';
```

Comment:
Note: some tests already exist in filterKeysSchema-test.ts (they test the global generation of the filterKeySchema)